### PR TITLE
Use generated object doc in the man page

### PR DESCRIPTION
**Pull request by t-wissmann**
Use the asciidoc file generated by format-doc.py in herbstluftwm.txt. To
make it fit into the man page and to make the include mechanism work
also for the website, a few adjustments were necessary.

Unfortunately, if a include fails, then asciidoc does not fail but only
prints a warning.

### CODE REVIEW

File: doc/CMakeLists.txt [hunk 1]
```diff
@@ -57,10 +74,13 @@ if (WITH_DOCUMENTATION)
     gen_manpage(herbstluftwm 1)
     gen_manpage(herbstluftwm-tutorial 7)
 
+
```
Comment (blarz) [inline doc/CMakeLists.txt:77]: Superfluous newline?

File: doc/format-doc.py [hunk 2]
```diff
@@ -199,21 +198,15 @@ def run(self, clsname, path=[]):
 def main():
     parser = argparse.ArgumentParser(description='Generate object documentation')
     parser.add_argument('jsondoc', help='the hlwm-doc.json file')
+    parser.add_argument('--objects-intro',
```
Comment (blarz) [inline doc/format-doc.py:201]: Is this parameter used anywhere? I cannot find it

**Review by blarz — CHANGES_REQUESTED** [review]

File: doc/CMakeLists.txt (hunk 1 above)
Comment (t-wissmann) [reply doc/CMakeLists.txt]: [x] removed.

File: doc/format-doc.py (hunk 2 above)
Comment (t-wissmann) [reply doc/format-doc.py]: That's a leftover from a previous approach. I've removed it.

Comment (blarz) [reply doc/format-doc.py]: Thanks!

**Review by blarz — APPROVED** [review]

**Pull request merged**